### DISCUSSION TITLE
Error: Config file was not found at "/vercel/path0/.vercel/output/config.json"

### DISCUSSION THREAD

**Discussion by seungyeah** [vercel/community#3650] [Help]
### Summary

Running build in San Francisco, USA (West) – sfo1
--
22:18:54.524 | Cloning completed: 861.451ms
22:18:55.189 | Using prebuilt build artifacts...
22:18:55.194 | Error: Config file was not found at "/vercel/path0/.vercel/output/config.json"
22:18:55.194 | at D6 (/var/task/sandbox.js:239:2647)
22:18:55.195 | at async y1e (/var/task/sandbox.js:248:11522)
22:18:55.195 | at async bmt (/var/task/sandbox.js:264:1900)
22:18:55.195 | at async ymt (/var/task/sandbox.js:264:1701)

I encountered this problem when I deployed my project.
I don't know why this error occurs.
I am using Svelte for my frontend and Rust for the backend.
This project runs and builds well in local.

### Steps to Reproduce

I don't know...

**Comment by alexboii** [comment]
Have had lots of similar errors in the past using Nextjs
